pypdf/_writer.py

```diff
@@ -872,7 +923,16 @@ def _update_text_field(self, field: DictionaryObject) -> None:
             else:
                 # Td is a relative translation
                 ap_stream += f"0 {- font_height * 1.4} Td\n".encode()
-            ap_stream += b"(" + str(line).encode("UTF-8") + b") Tj\n"
+            enc_line: List[bytes] = [
+                font_full_rev.get(c, c.encode("utf-16-be")) for c in line
+            ]
+            if any(len(c) >= 2 for c in enc_line):
+                ap_stream += b"<" + (b"".join(enc_line)).hex().encode() + b"> Tj\n"
```

Comment:
Codecov  reports that this line is never evaluated in `pypdf` test suite:
do you think a unit test could be added to cover this case?